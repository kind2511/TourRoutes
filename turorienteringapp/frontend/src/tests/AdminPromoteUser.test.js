import React from 'react';
import { render } from '@testing-library/react';
import PromoteUserButton from '../components/AdminPromoteUser';

test('PromoteUserButton renders without crashing', () => {
  render(<PromoteUserButton userId={1} onPromote={() => {}} />);
});

//---------------------------------->

// Snapshot test for the PromoteUserButton component
test('PromoteUserButton component snapshot', () => {
  const { asFragment } = render(<PromoteUserButton userId={1} onPromote={() => {}} />);
  expect(asFragment()).toMatchSnapshot();
});