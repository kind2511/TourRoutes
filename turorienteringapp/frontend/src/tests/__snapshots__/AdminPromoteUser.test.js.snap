// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PromoteUserButton component snapshot 1`] = `
<DocumentFragment>
  <button
    class="admin-users-promote-button"
  >
    Promote
  </button>
</DocumentFragment>
`;
